docs/classic-ui/index.md

```diff
@@ -10,7 +10,9 @@ html_meta:
 
 # Classic UI
 
-```{todo}
+```{admonition} TODO
+:class: warning
+
```

Comment:
I don't think any of these changes to MyST markup are necessary. The current MyST markup generates the following HTML:

```html
<div class="admonition-todo admonition" id="id1">
<p class="admonition-title">Todo</p>
<p>Perhaps some of this introduction should be an include, so we don't have to repeat ourselves?</p>
</div>
```

Here is the default.

![Screen Shot 2022-01-30 at 5 48 44 PM](https://user-images.githubusercontent.com/102112/151728418-3602fdad-41a4-431e-b223-7be2db7b0e09.png)

We could change the class `admonition-todo` to something that implies todo-ness.

![Screen Shot 2022-01-30 at 6 19 38 PM](https://user-images.githubusercontent.com/102112/151730443-c7d45a31-c6e6-4a6b-bc05-86d9510b412a.png)

## `docs/_static/custom.css`
```css

/* Add Font Awesome 5 icon for todo */
:root {
  --pst-icon-clipboard-list: '\f46d';
  --pst-icon-admonition-todo: var(--pst-icon-clipboard-list);
}

.admonition > .admonition-todo:before, div.admonition-todo > .admonition-title:before {
  content: var(--pst-icon-admonition-todo);
}
```

Please let me know. Thank you!